### PR TITLE
Update rhel8/fedora signatures for cobbler 2.8

### PR DESCRIPTION
**Pull request by kwizart**
Signed-off-by: Nicolas Chauvet <kwizart@gmail.com>

### CODE REVIEW

File: config/distro_signatures.json [hunk 1]
```diff
@@ -273,6 +289,38 @@
...
+    "supported_repo_breeds":["rsync", "rhn", "yum"],
+    "kernel_file":"vmlinuz(.*)",
+    "initrd_file":"initrd(.*)\\.img",
+    "isolinux_ok":false,
+    "default_kickstart":"/var/lib/cobbler/kickstarts/sample_end.ks",
+    "kernel_options":"repo=$tree",
+    "kernel_options_post":"",
+    "boot_files":[]
+   },
+   "fedora30": {
```
Comment (SchoolGuy) [inline config/distro_signatures.json]: You opened a seperate PR for this one. Was it intended that you doubled the PR or was this a mistake?

Comment (kwizart) [reply config/distro_signatures.json]: This PR is dedicated to the release28 branch over the master branch (cobbler 3.0)

Comment (kwizart) [reply config/distro_signatures.json]: thx for the review.

Comment (SchoolGuy) [reply config/distro_signatures.json]: Oh I should have taken a closer look of couse then this is even more lgtm.

**Comment by hendrahendra** [comment]
I'm getting this exception error with "release28":

```
$ cobbler import --name=RHEL8.0 --arch=x86_64 --path=/mnt/iso
task started: 2019-06-28_012207_import
task started (id=Media import, time=Fri Jun 28 01:22:07 2019)
Found a candidate signature: breed=redhat, version=rhel8
Found a matching signature: breed=redhat, version=rhel8
Adding distros from path /var/www/cobbler/ks_mirror/RHEL8.0-x86_64:
creating new distro: RHEL8.0-x86_64
Exception occured: <class 'cobbler.cexceptions.CX'>
Exception value: 'cobbler trigger failed: /var/lib/cobbler/triggers/change/post-trigger returns 1'
Exception Info:
  File "/usr/lib/python2.7/site-packages/cobbler/remote.py", line 82, in run
    rc = self._run(self)
   File "/usr/lib/python2.7/site-packages/cobbler/remote.py", line 227, in runner
    self.logger
   File "/usr/lib/python2.7/site-packages/cobbler/api.py", line 922, in import_tree
    return import_module.run(path,mirror_name,network_root,kickstart_file,arch,breed,os_version)
   File "/usr/lib/python2.7/site-packages/cobbler/modules/manage_import_signatures.py", line 133, in run
    os.path.walk(self.path, self.distro_adder, distros_added)
   File "/usr/lib64/python2.7/posixpath.py", line 246, in walk
    walk(name, func, arg)
   File "/usr/lib64/python2.7/posixpath.py", line 246, in walk
    walk(name, func, arg)
   File "/usr/lib64/python2.7/posixpath.py", line 238, in walk
    func(arg, top, names)
   File "/usr/lib/python2.7/site-packages/cobbler/modules/manage_import_signatures.py", line 248, in distro_adder
    adtls.append(self.add_entry(dirname,kernel,initrd))
   File "/usr/lib/python2.7/site-packages/cobbler/modules/manage_import_signatures.py", line 325, in add_entry
    self.distros.add(distro,save=True)
   File "/usr/lib/python2.7/site-packages/cobbler/collection.py", line 371, in add
    utils.run_triggers(self.api, ref, "/var/lib/cobbler/triggers/change/*", [], logger)
   File "/usr/lib/python2.7/site-packages/cobbler/utils.py", line 958, in run_triggers
    raise CX(_("cobbler trigger failed: %(file)s returns %(code)d") % { "file" : file, "code" : rc })

!!! TASK FAILED !!
```

**Comment by kwizart** [comment]
@hendrahendra Please report a separate issue with details on your setup.
This might not be directly related to this PR, let's investigate on a separate ticket.
Thx

**Comment by mjm455** [comment]
Has this been resolved because I'm also trying to get rhel8 added to cobbler but doing a "cobbler signature update" overwrites the json file and it doesn't have rhel8 in its list.  I was able to install release28 which has rhel8 but the update just downgrades that file.

**Comment by SchoolGuy** [comment]
@mjm455 The reason for this is because the github.io website for cobbler is not serving the most current file. Since this PR is merged I will update the signature file asap. But it will take me some hours.

**Comment by mjm455** [comment]
@SchoolGuy Thanks!  Looking forward to using Cobbler and Terraform with Redhat 8.0.

**Pull request merged**